src/components/ViewerMessaging/MessagingArgument.tsx

```diff
@@ -73,7 +121,7 @@ export function listProperties(definition: Definition, schema: MessageSchema) {
                     return (
                         <div key={propName} className="margin-bottom--md">
                             <div className="margin-bottom--sm">
-                                <code>{propName}</code>
+                                <h4>{propName}</h4>
```

Comment:
Can we preserve the `<code>` tag inside the `<h4>`?

Reply:
I made a fake header style for this so that it can be monospaced.